stories/notifications.stories.js

```diff
@@ -205,4 +208,20 @@ storiesOf( 'Global/Notifications', module )
 			}
 			type="win-stats"
 		/>
-	) );
+	) )
+	.add( 'User Input Settings', () => {
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( CORE_SITE ).receiveSiteInfo( {
+				usingProxy: true,
+				referenceSiteURL: 'http://example.com',
+				adminURL: 'http://example.com/wp-admin',
+				siteName: 'My Site Name',
+			} );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<UserInputSettings />
```

Comment:
One last problem here – the image displayed within it doesn't load because the webpack config for Storybook is adding a hash to the file URL (also we don't use the `media` directory, it seems to be a default used by Storybook?):

```
GET http://localhost:49638/assets/static/media/person-sit.08eab5f1.png 404 (Not Found)
```
![image](https://user-images.githubusercontent.com/1621608/99517277-d842f780-2997-11eb-9361-b0c60e5eb4c2.png)